### PR TITLE
Fix reading prescale values during miniAOD production

### PR DESCRIPTION
**Pull request by gpetruc**
Fix reading prescale values during miniAOD production by setting ReadPrescalesFromFile = False, to get the proper PS column
https://hypernews.cern.ch/HyperNews/CMS/get/physTools/3572/1/1.html

@arizzi @rekovic

### CODE REVIEW

**Comment by gpetruc** [comment]
please test

File: PhysicsTools/PatAlgos/python/triggerLayer1/triggerProducer_cfi.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 ,l1GtTriggerMenuLiteInputTag = cms.InputTag("gtDigis")
 ,l1tAlgBlkInputTag = cms.InputTag("gtStage2Digis")
 ,l1tExtBlkInputTag = cms.InputTag("gtStage2Digis")
-,ReadPrescalesFromFile = cms.bool(True)
+,ReadPrescalesFromFile = cms.bool(False)
```
Comment (slava77) [inline PhysicsTools/PatAlgos/python/triggerLayer1/triggerProducer_cfi.py]: @rekovic 
please clarify why this was originally added with "True".

Is this solution (using "False") going to work for MC as well as for data?

**Comment by slava77** [comment]
@cmsbuild please test

nanoAOD harvesting is now available, I'm curious to see the comparisons.

File: PhysicsTools/PatAlgos/python/triggerLayer1/triggerProducer_cfi.py (hunk 1 above)
Comment (slava77) [reply PhysicsTools/PatAlgos/python/triggerLayer1/triggerProducer_cfi.py]: @rekovic 
please comment
Thank you.

**Comment by slava77** [comment]
assign l1

we need feedback from L1 on correctness of using the configuration parameters

**Comment by rekovic** [comment]
@slava77 
> please clarify why this was originally added with "True".

This was the original configuration before the prescales were available from DB.

**Comment by rekovic** [comment]
+1

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/22131	1156607
- start using DB for prescale info in miniAOD
- jenkins tests pass and comparisons show no differences

**Comment by fabiocos** [comment]
+1

**Comment by fabiocos** [comment]
merge

**Pull request merged**